OWNERS_ALIASES

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs: https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md
+
+aliases:
+#  sig-cluster-lifecycle-leads:
+  cluster-api-admins:
+    - kris-nova
+  cluster-api-maintainers:
+    - kris-nova
+  cluster-api-do-maintainers:
+    - andrewsykim
+    - alvaroaleman
```

Comment:
@alvaroaleman To be able to lgtm when this is moved upstream, you'll need to request for membership for kubernetes and kubernetes-sigs here: https://github.com/kubernetes/org/issues/new?template=membership.md.
More details about membership - https://github.com/kubernetes/community/blob/master/community-membership.md#member.

I'll gladly +1 on the request, but the second sponsor needs to be from a different company (pretty sure anyone would happily +1 because of your contributions in cluster-api :))

Reply:
To be clear, this is not a blocker. Let's merge it here! :tada: